File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
     if (x < 0 || (x % 10 === 0 && x !== 0)) {
        return false;
    }

    let string=x.toString()
    let reverce=string.split("").reverse().join("")
  return reverce===string;
}

File: 0009-palindrome-number/0009-palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
      if(x<0){
        return false;
      }
      string s=x.ToString();
      for(int i=0;i<s.Length/2;i++){
        if(s[i]!=s[s.Length-1-i]){
            return false;
        }
      }
      return true;
    }
}